src/domains/Session/SessionService.php

```diff
@@ -18,6 +18,10 @@
  */
 class SessionService
 {
+    const PASSWORD_HASH_ALGORITM = PASSWORD_BCRYPT;
```

Comment:
Not sure if this is >PHP7, else use private const.

Reply:
What, assigning a constant with a constant? That works on 5.6.

Reply:
No I mean, I'm not sure if Mimoto has a >PHP7 requirement. If so you can use the new `private const` feature 😛

Reply:
Ah sorry, misread that part. There's at least no php requirement in the composer.json, so I assume 7 is not enforced.

Reply:
Yeah I saw that as well so I added this ticket:
https://github.com/SebastianKersten/Mimoto/issues/15

Reply:
Ah, minimum php version is 5.6 so this is closed.